src/Ledger/Conway/Specification/Enact/Properties/Computational.lagda.md

```diff
@@ -19,6 +19,21 @@ open EnactState
 
 open Computational ⦃...⦄
 
+module _ {Γ : EnactEnv} {eSt : EnactState} {ga : GovAction} where
+
+  -- TODO: Make this follow from computationalness or viceversa
```

Comment:
I just happened to see this PR, we actually have a lemma for this already: `computational⇒rightUnique` in `Interface.ComputationalRelation`.